sky/clouds/gcp.py

```diff
@@ -259,7 +262,8 @@ def make_deploy_resources_variables(
                     image_id = _IMAGE_ID_PREFIX + 'common-cu113-v20220701'
                 else:
                     # CUDA driver version 510.47.03, CUDA Library 11.6
-                    image_id = _IMAGE_ID_PREFIX + 'common-cu113-v20220806'
+                    # Does not support torch==1.13.0 with cu117
+                    image_id = _IMAGE_ID_PREFIX + 'common-cu113-v20221026'
```

Comment:
Does the `cu113` string indicate it's 11.3 not 11.6 as L264 suggested? https://cloud.google.com/deep-learning-vm/docs/images#listing-versions

Same for L262

Reply:
Worth a comment!